### PR TITLE
bump uikit version - 1.10.3

### PR DESCRIPTION
**Pull request by caravinci**

### CODE REVIEW

File: uikit/PercentageBar/index.tsx [hunk 1]
```diff
@@ -66,7 +66,8 @@ const PercentageBar = ({
       `}
     >
       <VAlignedText>
-        {nom.toLocaleString()} <Icon name="slash" height="15px" fill="grey_2" /> {denom.toLocaleString()}
+        {nom.toLocaleString()} <Icon name="slash" height="15px" fill="grey_2" />{' '}
```
Comment (ciaranschutte) [inline uikit/PercentageBar/index.tsx:69]: space added here?

Comment (caravinci) [reply uikit/PercentageBar/index.tsx:69]: lol must've been some prettier styling when I pressed save. want me to remove it?

Comment (ciaranschutte) [reply uikit/PercentageBar/index.tsx:69]: yes please. seems minor but I could ee how this on a percentage icon causes _chaos_

**Comment by ciaranschutte** [comment]
looks good on `platform-ui` and `storybook`

**Review by ciaranschutte — APPROVED** [review]

**Pull request merged**